front/app/containers/IdeasShow/components/RightColumnDesktop/index.tsx

```diff
@@ -99,25 +93,28 @@ const RightColumnDesktop = ({
                 <Divider />
               </>
             )}
-            {/* Participation method is 'ideation' */}
-            {showIdeaReactionControl && (
-              <>
-                <ReactionControl styleType="shadow" ideaId={ideaId} size="4" />
-                <Divider />
-              </>
-            )}
-            {/* Participation method is 'voting' */}
-            {/* TODO: Fix this the next time the file is edited. */}
-            {/* eslint-disable-next-line @typescript-eslint/no-unnecessary-condition */}
-            {phase && ideaIsInParticipationContext && votingConfig && (
-              <Box pb="24px" mb="24px" borderBottom="solid 1px #ccc">
-                {votingConfig.getIdeaPageVoteInput({
-                  ideaId,
-                  phase,
-                  compact: false,
-                })}
-              </Box>
-            )}
+            {participationMethod === 'ideation' &&
```

Comment:
We indirectly already checked for this participationMethod in showIdeationReactions before:
- This component only gets rendered if participation method is `ideation`, `proposals`, or `voting`.
- We returned false in showIdeaReactionControl if the PM was  not `proposals` or `voting`.

This is much clearer, IMO (although not as easy to reuse as it could be).